person.rb

```diff
@@ -1,16 +1,19 @@
-class Person
-  attr_reader :id
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id, :correct_name
```

Comment:
- [ ] As mentioned by the previous reviewer, As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md#:~:text=Make%20sure%20that%20this%20class%20has%20a%20method%20correct_name%20implemented.%20It%20should%20simply%20return%20the%20name%20attribute), Please make sure that person class has a method `correct_name` implemented, not getters for it only.
[Hint] It will be something like that:
```ruby
def correct_name
       name
end

```

Reply:
:correct_name is the same thing as the code you are asking me to write

If I do that, rubocob will still remove it, because I'm just writing the same thing

Reply:
I actually did

Reply:
You can remove `:correct_name` from line 4 and leave the method you created.